Modules/user/Resources/views/user-settings/index.blade.php

```diff
@@ -0,0 +1,38 @@
+@extends('user::layouts.master')
+
+@section('content')
+
+<form action ="{{ route('user.settings') }}" method = "POST">
+@csrf
+<div class="container">
+    <div class="row justify-content-around">
+        <div class="col-6">
+            <label class="user-settings" style="font-size:24px;">User Settings</label>
+        </div>
+        <div class=".col-6 .offset-md-3">
```

Comment:
`.` is not used inside class attribute. Please make these changes and also check if the UI is affected due to this change or not.
```suggestion
        <div class="col-6 offset-md-3">
```

Reply:
Removing dots from div are affecting UI

Reply:
Then remove both classes as they are not being used